### DISCUSSION TITLE
I am stuck :( Time recording could not be stopped There was a problem saving. Maximum 10:00 hours allowed.

### DISCUSSION THREAD

**Discussion by gerroon** [kimai/kimai#4298] [Questions & Answers] [answered]
I cant stop the time, it keeps giving me this error. How can I stop the timer, I am passed 11 hours? This sounds like a catch22 situation

**Comment by kevinpapst** [comment]
https://www.kimai.org/documentation/timesheet.html#stop-records

**Reply by gerroon** [reply]
Thanks for the reply. I was able to stop it by editing the entry but I do not see the settings mentioned in the doc.

**Reply by kevinpapst** [reply, marked as answer]
<img width="838" alt="Bildschirmfoto 2023-09-16 um 23 35 31" src="https://github.com/kimai/kimai/assets/533162/a59fa11b-494a-4820-9978-63227fd903cc">
